### DISCUSSION TITLE
Check whether a collection is loaded with Milvus 1.x

### DISCUSSION THREAD

**Discussion by hadim** [milvus-io/milvus#15292] [Q&A and General discussion]
Is that possible to check (ideally from the python milvus client) whether a given collection is loaded in memory and so ready to be used?

**Comment by yhmo** [comment]
No api to do that.
When you call the load_collection(), milvus will evict other data from the cache and try to load all segments of the collection into the cache.
If the cache size is not enough to hold all segments, only some of the segments are in the cache, but the load_collection() doesn't report error.
